scripts/fix-browser-order.js

```diff
@@ -53,52 +53,4 @@ const fixBrowserOrder = filename => {
   }
 };
 
-if (require.main === module) {
```

Comment:
What's going on with this removal?

Reply:
This file isn't really intended to be run on its own, so that's why I've removed this clause.